test/render/gltf-test-cases.js

```diff
@@ -0,0 +1,26 @@
+import {AnimationLoop} from '@luma.gl/core';
+
+const examples = {
+  avocado: require('../../examples/gltf/app').avocado,
+  milkTruck: require('../../examples/gltf/app').milkTruck
+};
+
+export default Object.keys(examples).map(name => {
+  const app = examples[name];
+  const animationLoop = new AnimationLoop(app);
+  let count = 0;
+  return {
+    name,
+    onInitialize: params => {
+      return animationLoop.onInitialize(params);
+    },
+    onRender: params => {
+      animationLoop.onRender(params);
+      if (count++ > 2000) {
```

Comment:
@Pessimistress I need a way to wait here till textures load...

Reply:
- See the discussion at https://github.com/uber/luma.gl/pull/911#discussion_r257047484
- You can use `params.time` or `params.tick` for a hard-coded wait period